cob_cartesian_controller/package.xml

```diff
@@ -13,25 +12,17 @@
   <license>LGPL</license>
 
   <buildtool_depend>catkin</buildtool_depend>
-  
-  <build_depend>cmake_modules</build_depend>
-  <build_depend>cob_srvs</build_depend>
-  <build_depend>geometry_msgs</build_depend>
-  <build_depend>roscpp</build_depend>
-  <build_depend>roslib</build_depend>
-  <build_depend>std_msgs</build_depend>
-  <build_depend>std_srvs</build_depend>
-  <build_depend>tf</build_depend>
-  <build_depend>visualization_msgs</build_depend>
 
-  <run_depend>cob_srvs</run_depend>
-  <run_depend>geometry_msgs</run_depend>
-  <run_depend>roscpp</run_depend>
-  <run_depend>roslib</run_depend>
-  <run_depend>rospy</run_depend>
-  <run_depend>std_msgs</run_depend>
-  <run_depend>std_srvs</run_depend>
-  <run_depend>tf</run_depend>
-  <run_depend>visualization_msgs</run_depend>
+  <depend>cmake_modules</depend>
+  <depend>cob_srvs</depend>
+  <depend>geometry_msgs</depend>
+  <depend>roscpp</depend>
+  <depend>roslib</depend>
+  <depend>std_msgs</depend>
+  <depend>std_srvs</depend>
+  <depend>tf</depend>
+  <depend>visualization_msgs</depend>
+
+  <exec_depend>rospy</exec_depend>
```

Comment:
In the restructured `cob_cartesian_controller` python will be used (PR still to be issued)...keeping `rospy` for now...